doc/thirdPartyRunPlugins.md

```diff
@@ -0,0 +1,35 @@
+# Third-Party Run Plugins
+
+Collection of third-party Run plugins created by the community that aren't distributed with PowerToys.
+
+## Installing / Upgrading
+
+The recommended path for the plugins is `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`.
+
+- Close PowerToys
+- Copy the plugin folder to `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`
+- Delete previous version in case of upgrade
+- Open PowerToys
```

Comment:
```suggestion
1. Close PowerToys.
1. Copy the plugin folder to `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Plugins`.
1. If you're upgrading a plugin, delete the previous version.
1. Open PowerToys.
```

Step-by-step instructions [should be in a numbered list and each step should end in a period](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions#complex-procedures).